contentcuration/contentcuration/frontend/settings/pages/Storage/RequestForm.vue

```diff
@@ -78,10 +78,11 @@
             v-if="item.license_url"
             class="mt-1"
           >
-            <ActionLink
+            <KExternalLink
+              class="link-css notranslate"
```

Comment:
```suggestion
              class="license-link"
```

- Links are translated
- A bit more clear naming